### DISCUSSION TITLE
ComfyUI support needed

### DISCUSSION THREAD

**Discussion by ultimatech-cn** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#306] [Ideas]
Is there any plan for comfyUI custom node support?

**Comment by timothyportnoff** [comment]
Bump. Also came here because I was able to install it through the ComfyUI Add-On manager but I'm stuck on how to continue :/

**Comment by Vigilence** [comment]
Still hoping for comfyu support from the author.
